.gitignore

```diff
@@ -8,6 +8,8 @@ pkg
 spec/.straight/default_last_keychain_id
 spec/.straight/second_gateway_last_keychain_id
 spec/.straight/straight.db
+spec/.straight/addons.yml
+spec/.straight/straight.log
```

Comment:
Why are these added to gitignore? Those files where supposed to have test-specific settings. Are they somehow created automatically now? Looks like that's not the case because Travis CI build is failing for this very reason: those files are absent.